.eslint-rules/helpers/index.js

```diff
@@ -0,0 +1,50 @@
+const fs = require("fs");
+
+const buildPathGroupsBasedOnWebpackAliases = ({
+  customJSRoot = "app/javascript/",
+  customAliasPath = "config/webpack/resolve.js",
+}) => {
+  const rootOfProject = __dirname + `/../../`;
+
+  const isFile = filePath =>
+    fs.existsSync(filePath) && fs.lstatSync(filePath).isFile();
+
+  const webpackAliasPath = rootOfProject + customAliasPath;
+
+  const hasWebpackAliasConfig = isFile(webpackAliasPath);
+
+  const isRailsProject = isFile(rootOfProject + "Gemfile");
+
+  const emptyPathGroups = [];
+
+  if (!hasWebpackAliasConfig || !isRailsProject) return emptyPathGroups;
+
+  const { alias } = require(webpackAliasPath);
+
+  const railsJSFilesRoot = rootOfProject + customJSRoot;
+
+  const pathGroups = Object.entries(alias).map(([name, path]) => {
+    // sometimes alias might be already resolved to full absolute path
+    const isAleadyAnAbsolutePath =
```

Comment:
Should I remove cypress and neetoUI related setup?

Reply:
Why aren't you using common configuration from `neeto-commons-frontend/configs` just like all other packages?